packages/cli/src/Generate.ts

```diff
@@ -136,11 +134,6 @@ ${bold('Examples')}
         printDownloadProgress: !watchMode,
         version: enginesVersion,
         cliVersion: pkg.version,
-        dataProxy:
-          !!args['--data-proxy'] ||
-          !!args['--accelerate'] ||
-          !!process.env.PRISMA_GENERATE_DATAPROXY ||
-          !!process.env.PRISMA_GENERATE_ACCELERATE,
```

Comment:
I think we could update the docs at https://www.prisma.io/docs/reference/api-reference/environment-variables-reference#prisma_generate_dataproxy

To say that this env var was removed in `5.2.0`

Reply:
Actually, I will restore it via `--no-engine`. I think it is fair to assume that if you have such an env var, you'd also want the engine not to be included otherwise it would be a breaking change.